lib/idpartner.js

```diff
@@ -170,16 +171,16 @@ class IDPartner {
       code_challenge: codeChallenge,
       state,
       nonce,
-      scope: scope.join(' '),
-      prompt,
+      scope: scopeString,
       response_type: 'code',
       client_id: clientId,
       nbf: Math.floor(new Date().getTime() / 1000),
       'x-fapi-interaction-id': uuidv4(),
       identity_provider_id: query.idp_id,
       idpartner_token: idpartnerToken,
-      claims: JSON.stringify(claimsObject),
       response_mode: 'jwt',
+      claims: JSON.stringify(claims),
+      ...otherAuthorizationParams,
```

Comment:
you are gonna overwrite claims

Reply:
Yup, is this an action item ?

Reply:
I see, no, I extract the claims up in line 151. And, the data in `otherAuthorizationParams` are the remnants of that decronstruction

Reply:
sweet!